docs/scaling.md

```diff
@@ -98,18 +103,33 @@ The following are the steps to increase the size:
     ```
 
 4. In the CR configuration file, use vim or another text editor to edit
-    the PVC size.
+    the PVC size. 
 
     ```{.bash data-prompt="$"}
     $ vim CR_backup.yaml
     ```
 
-5. Apply the updated configuration to the cluster.
+5. Set the new storage size for the `mysql.volumeSpec.persistentVolumeClaim.resources.requests.storage` option:
+
+    ```yaml
+    mysql:
+    volumeSpec:
+      persistentVolumeClaim:
+        resources:
+          requests:
+            storage: 6Gi
+    ```
+
+    The size of the storage must match the size you defined for the PVC object on the cluster nodes.
+
+6. Apply the updated configuration to the cluster.
 
     ```{.bash data-prompt="$"}
     $ kubectl apply -f CR_backup.yaml
     ```
 
+The storage size change takes some time. When it starts, the Operator automatically adds the `pvc-resize-in-progress` annotation to the `PerconaServerMySQL` Custom Resource. The annotation contains the timestamp of the resize start and indicates that the resize operation is running. After the resize finishes, the Operator deletes this annotation.
```

Comment:
this is wrong at the moment, automated storage scaling will be introduced in v0.11.0

Reply:
I hide it till 0.11.0